pandas/io/parsers.py

```diff
@@ -101,7 +100,8 @@ class ParserWarning(Warning):
 keep_default_na : bool, default True
     If na_values are specified and keep_default_na is False the default NaN
     values are overridden, otherwise they're appended to
-parse_dates : boolean, list of ints or names, list of lists, or dict, default False
+parse_dates : boolean, list of ints or names, list of lists, or dict,
+    default False
```

Comment:
Same comment as in one of the previous rounds. This 'default False' on the new line will give non-ideal results in the html docs.